documentation/src/getting_started.md

```diff
@@ -1,19 +1,100 @@
 # Getting started
 
-Getting started with MomentoBooth is easy! You only need a computer/laptop with webcam. Specifically, follow these steps:
```

Comment:
I get the new information, but would not remove a friendly welcome to the page.

Reply:
Added back something similar and moved the warnings to the end of the page